src/Modules/Cart/CartEndpoint.php

```diff
@@ -0,0 +1,104 @@
+<?php namespace Lean\Woocommerce\Modules\Cart;
```

Comment:
Also I think the `Modules` does not have value for naming and clarification if for instance you change this to: 

```php
 <?php namespace Lean\Woocommerce\APi;

class Cart {
}
```

We can say that class `Cart`  is part of the `API` such as `Lean\Woocommerce\Api\Cart;`

So with this change you avoid having to add the suffix `Endpoint` on other classes that are specific for endpoints.